src/app/selectors.ts

```diff
@@ -113,8 +113,7 @@ export const skipVerificationSelector = (state: RootState) => state.app.skipVeri
 
 export const fiatConnectCashInEnabledSelector = (state: RootState) =>
   state.app.fiatConnectCashInEnabled
-export const fiatConnectCashOutEnabledSelector = (state: RootState) =>
-  state.app.fiatConnectCashOutEnabled
+export const fiatConnectCashOutEnabledSelector = (state: RootState) => true
```

Comment:
for some reason `state.app.fiatConnectCashInEnabled` was false for me. I assume this had something to do with a local dev build.